ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec-nonprod.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
Do we need this nonprod.json file?

Reply:
Removed this file as its not needed anymore